torch/_dynamo/symbolic_convert.py

```diff
@@ -3080,6 +3090,7 @@ def _return(self, inst):
             and not self.inconsistent_side_effects
             and not self.symbolic_locals_contain_module_class()
             and not self.export
+            and not self.one_graph
```

Comment:
This is the main one line change. Everything else is work to fix issues revealed by adding this line.